Update Get Started “searchgov-for-federalist”

dawnpm opened the pull request:
Automatically generated by Netlify CMS

dawnpm commented:
Noting here that https://github.com/cloud-gov/pages-uswds-jekyll/blob/main/README.md still refers to `npm run federalist`, so I'm leaving that one alone.

dawnpm commented:
Also noting here that because I opened this with Netlify CMS, it didn't give me the PR template.

amanifarooque commented:
@dawnpm could you post links to federalist preview of the URLs that changed?

Also, we'll need to update the config.yml with the correct sidenav link for Search.gov for Cloud.gov Pages instructions. Might be a good time to add this to the header menus as well?

amanifarooque commented:
Nevermind, was able to find it! I think redirects aren't working correctly with such a nested preview branch, but doesn't seem worth it to troubleshoot right now.

Cloud.gov Pages Instructions: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/searchgov-for-cloudgov-pages.html

Site Launch Guide: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/site-launch-guide.html 

I would recommend editing this in code rather than NetlifyCMS since you'll need to edit a few different non-.md files:

- config.yml to make the navigation updates mentioned above
- _includes/featured-accordion.html to update the indexing section's reference to Federalist

Lastly, some of the links are not displaying correctly. Make sure you are in "Markdown" not "Rich Text" mode in NetlifyCMS if you are adding links in markdown.

amanifarooque reviewed (changes requested):
See comments above for fixes needed.

dawnpm commented on get-started/searchgov-for-cloudgov-pages.md:
```diff
@@ -0,0 +1,57 @@
+## On the Search.gov side of things:
+
+1. Sign up for a [user account](https://search.usa.gov/signup).
+2. Read our \[Search Site Launch Guide]({{ site.baseurl }}/manual/site-launch-guide.html) if you’d like some direction.
```
NetlifyCMS was trying to kill me. It kept jumping the cursor to the end of the paragraph after each character I typed. I switched to richtext mode hoping it would be different, but it was not. Any idea why it might have been doing that?

amanifarooque replied:
Not sure, I would have to investigate. Can you make these changes via code for now? I can make an item in my backlog to retest netlifyCMS.

dawnpm replied:
Yeah, I bailed out to local after doing just one file.

amanifarooque replied:
seems like a known bug - https://github.com/netlify/netlify-cms/issues/5092 will try the recommended CSS fix

dawnpm commented:
Looks like my previous find-all was looking only in markdown files. Have updated the sidenav, added "Cloud.gov Pages Integration" to the main navigation, and updated the indexing accordion.

I think the redirects never work in preview because they are relative.

Cloud.gov Pages previews:
https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/searchgov-for-cloudgov-pages.html

https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/indexing/indexing-workflow.html

https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/site-launch-guide.html

No preview for the readme, pr checklist, or the in-comment changes to the status page.

dawnpm commented:
@amanifarooque ready for re-review.

dawnpm commented:
@amanifarooque which requested change is outstanding?

amanifarooque commented:
@dawnpm The numbering issue, see above.

dawnpm commented:
@amanifarooque which issue is that? I don't see any mention of numbers or numbering, even using ctrl-F in the source of this page. thanks!

amanifarooque commented:
@dawnpm You should be able to see my comment now, didn't realize it wouldn't post until I officially added another review.

Preview Page: https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/searchgov-for-cloudgov-pages.html

The numbering is not consistent. 
<img width="750" alt="image" src="https://user-images.githubusercontent.com/7243053/198686570-4cdd75bc-b2d3-464e-a4f8-07f7224349eb.png">

<img width="759" alt="image" src="https://user-images.githubusercontent.com/7243053/198686583-fc0b9a17-ec9f-49aa-8903-6c23d40207ed.png">

dawnpm commented:
@amanifarooque finally ready for re-review, I had to convert the list to html to get the numbering to behave. 

https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/cms/get-started/searchgov-for-federalist/get-started/searchgov-for-cloudgov-pages.html

amanifarooque commented on get-started/searchgov-for-cloudgov-pages.md:
```diff
@@ -0,0 +1,64 @@
+<li>Go to the Search.gov Admin Center, and find the <code>Activate</code> section, <code>API Access Key</code>. Copy this value to the <code>access_key</code>line in your Cloud.gov Pages <code>_config.yml</code>file.</li>
+</ul>
+</li>
+<li>If you do not want [type-ahead search suggestions]({{ site.baseurl }}/manual/typeahead-api.html) to appear in your website’s search box, change the <code>suggestions</code>value to <code>false</code>.</li>
```
Markdown formatting needs to be changed to an html a tag

dawnpm replied:
crud, missed that one. fixed.

amanifarooque reviewed (changes requested):
One remaining tweak - a markdown link needs to be changed to HTML formatting

amanifarooque commented on get-started/searchgov-for-cloudgov-pages.md:
```diff
@@ -0,0 +1,64 @@
+<li>If you do not want [type-ahead search suggestions]({{ site.baseurl }}/manual/typeahead-api.html) to appear in your website’s search box, change the <code>suggestions</code>value to <code>false</code>.</li>
+<li>When you're ready to launch your website, you will uncomment  the <code>url</code>line, to allow your XML sitemap to build using your production domain. Then, let our team know you're live and we'll begin indexing.<br />
+<ul>
+<li>If you tested using a demo domain, you'll need to update the <code>url</code>value to be your production domain.</li>
```
Need spaces after the </code> tag
<img width="708" alt="image" src="https://user-images.githubusercontent.com/7243053/201171406-a83ba786-4568-4b3e-a3dc-0f16e2cb681e.png">

amanifarooque commented on get-started/searchgov-for-cloudgov-pages.md:
```diff
@@ -0,0 +1,64 @@
+<ol><li>Open the <code>_config.yml</code> file in your Cloud.gov Pages repo.</li>
+<li>Add your site's domain to the <code>url</code> field, approximately line 25. Leave this line commented out for now.<br />
+<ul>
+<li>If you're testing pre-production, enter your **demo** domain in the <code>url</code> field, and uncomment the line to allow the XML sitemap to build using this demo domain.</li>
```
** needs to be converted to <strong> tags

dawnpm commented:
@amanifarooque ready for re-review

amanifarooque reviewed (approved):
LGTM
